migrations/0032-add-created-at-column-to-merch-collection.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCreatedAtColumnToMerchCollection1640851101278 implements MigrationInterface {
+  name = 'AddCreatedAtColumnToMerchCollection1640851101278';
+
```

Comment:
nit: the `name` field here isn't necessary since it defaults to the class name, which is the same in this case. See https://github.com/typeorm/typeorm/blob/master/src/migration/MigrationInterface.ts#L10